extensions/unsplash/src/components/Actions.tsx

```diff
@@ -39,16 +33,14 @@ export const Sections: React.FC<BaseProps> = ({ details = false, item }) => {
   };
 
   return (
-    <>
+    <React.Fragment>
```

Comment:
Is there any specific reason why you use `React.Fragment` instead of the shorthand `<>`?

Reply:
no its just preference lol

Reply:
Would you mind changing them to the shorthand `<>`, if you don't want, it's fine 😊.